homeassistant/components/vicare/strings.json

```diff
@@ -3,7 +3,7 @@
     "flow_title": "{name} ({host})",
     "step": {
       "user": {
-        "title": "{name}",
+        "title": "Setup a new Viessmann device",
```

Comment:
Why does this have a title in the first place?

The description already provides the context, and if the title is omitted, the name of the integration will be shown.

I would suggest on removing it instead.

Reply:
done